src/main/java/io/github/bonigarcia/wdm/Architecture.java

```diff
@@ -23,7 +23,7 @@
  * @since 1.0.0
  */
 public enum Architecture {
-    DEFAULT, X32, X64;
+    DEFAULT, X32, X64, X7HF;
```

Comment:
The new architecture value should be ARMv7h instead.

Reply:
These hacks were made because firefox doesn't name the package correctly.